src/installer/managed/Microsoft.NET.HostModel/Bundle/TargetInfo.cs

```diff
@@ -91,17 +89,20 @@ public string GetAssemblyName(string hostName)
 
         public override string ToString()
         {
-            string os = IsWindows ? "win" : IsLinux ? "linux" : "osx";
+            string os = IsWindows ? "win" : IsLinux ? "linux" : IsFreeBSD ? "freebsd" : "osx";
```

Comment:
```suggestion
            string os = IsWindows ? "win" : OS.ToString().ToLowerInvariant();
```

Reply:
This is the last use of `IsFreeBSD` property. The property can be deleted if you accept this change.